perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -1884,17 +1885,33 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 * @param vo vo
 	 * @param query query with page information
 	 * @param attrNames attribute names
+	 * @param searchString (optional) get only members based on this string and specific parameters in DB,
```

Comment:
I would enhance this message, since its documentation for external API. Its useful to tell, that by default it searches through names, emails, logins of member or other attributes specified by the perun configuration.

Reply:
Ok, since I have moved the parameters to the MembersPageQuery, the message was removed. But I have added a note about query parameter to the method description.